### PR TITLE
Update Main.java

### PR DESCRIPTION
**Pull request by anileinchen**
changed mturk url

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java [hunk 1]
```diff
@@ -268,7 +268,7 @@ private static void initLogLevel(Config config) {
                     //FIXME remove the sandbox url - but I am to paranoid
                     platformInstance = mturkPlatform = new MturkPlatform(platform.user,
                             platform.password,
-                            "https://mechanicalturk.sandbox.amazonaws.com/",
+                            "https://mechanicalturk.amazonaws.com/",
```
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:271]: Hello,

i hardcoded this value that noone uses productive mturk by accident. there is platform.url which will take the value form the config file, which doesnt need a hardcoded string at all here. So platforms can still be used for productive OR sandbox.

**Comment by anileinchen** [comment]
Was just a little hack for me to test something. normally you are totally right. I will solve it in an other way.

**Comment by LeanderK** [comment]
hacks are useful for prototyping, but i would prefer to not have them in the master where possible. If you have any questions, just shoot us an email! we are happy to help.

**Comment by kelunik** [comment]
This should just use `platform.url` finally.

**Comment by LeanderK** [comment]
sometimes i wonder whether you are to idealistic for this kind of work ;) Yes it should, but it does not

**Pull request closed without merging**